presto-orc/src/main/java/com/facebook/presto/orc/ColumnGroupReader.java

```diff
@@ -724,58 +663,9 @@ private void alignResultsAndRemoveFromQualifyingSet(int numAdded, int lastStream
 
     private static int getCurrentRow(StreamReader reader)
```

Comment:
getCurrentRow method is used only once and it is now quite trivial; let's inline it